tests/cases/text/metrics.slint

```diff
@@ -6,6 +6,7 @@ component Text2 inherits  Text {}
 export component TestCase inherits Window {
     width: 100phx;
     height: 100phx;
+    default-font-size: 10px;
```

Comment:
Why is that needed?
Is that a change of behavior?

Reply:
I oddly enough, needed this, when running locally. Now that I tried again it passes now. I'll remove it again and then we can see what happens in the CI :)

Reply:
I foudn out again why I needed this, re-added it now, along with a comment.

Reply:
why was it working before this change?

Reply:
It was "working" before this change because the testing backend completely ignored the font request:

<img width="828" alt="Bildschirmfoto 2025-04-24 um 09 03 06" src="https://github.com/user-attachments/assets/bba34183-9b31-4e45-a45a-9e6a1d1ecd4e" />

Reply:
Ah indeed, nice!